Tests/Test_BSP_WDTimer.c

```diff
@@ -8,7 +8,7 @@ int main(){
     uint32_t delay = 150000000;
     uint32_t time = 40;
     uint32_t resetCount = 3;        //for testing WDTimer resets, these delays will let BSP_Reset() be called before the WDTimer finishes counting down 
-    BSP_WDTimer_Init();
+    BSP_WDTimer_Init(false);
```

Comment:
same thing here, should there be a comment for what value

Reply:
The comment is in the c file.